uploadNeuroDB/NeuroDB/HRRT.pm

```diff
@@ -303,16 +302,18 @@ sub read_matlab_file {
     my @matlab_files = grep ( /.m$/, @{ $self->{hrrt_files} } );
     my $matlab_file  = $matlab_files[0];
 
-    open my $fh, '<', $matlab_file;
-    chomp(my @info = <$fh>);
-    close $fh;
-
-    foreach my $line_nb ( @info ) {
-        next unless ( $line_nb =~ / = / );
-        my ($key, $val) = split( ' = ', $line_nb );
-        $val =~ s/;\cM$//;
-        # set header information
-        $self->{matlab_info}->{$key} = $val;
+    if (defined($matlab_file)) {
```

Comment:
I am curious, are there instances where there are no matlab files?

Reply:
If the script fail this block throws a warning (I think it failed before the folder hrrt_archive was created). So in most case the if case is not necessary but I find it more elegant to avoid warnings to be printed, even when things goes wrong.